site/content/integrate/apps/_index.md

```diff
@@ -1,34 +1,50 @@
 ---
 title: "Apps (Developers Preview)"
-heading: "Overview"
+heading: "Introduction"
 description: "Apps are lightweight interactive add-ons to Mattermost."
 section: "integrate"
 weight: 90
 ---
 
-Apps are lighweight interactive add-ons to Mattermost. Apps can:
+Apps are lightweight, interactive add-ons to Mattermost which use serverless hosting to run without any dedicated infrastructure. Apps can:
```

Comment:
That is not correct. Apps are not bound to serverless hosting. Any hosting mechanism for hosting an HTTP server works.

Reply:
```suggestion
Apps are lightweight, interactive add-ons to Mattermost which can use any HTTP-compatible hosting mechanism to run without dedicated infrastructure. Apps can:
```